ui/ts/models/cockroachlabs.ts

```diff
@@ -128,6 +128,7 @@ module Models {
             m.request<VersionList>({
               url: `${COCKROACHLABS_ADDR}/api/clusters/updates?uuid=${this.clusterID}&version=${nodeStatuses[0].build_info.tag}`,
               config: Utils.Http.XHRConfig,
+              background: true,
```

Comment:
Do you also need this on the other methods like `unregister`? It's still hanging on those.